config/defaults.go

```diff
@@ -80,6 +86,10 @@ func (d *Defaults) Validate() error {
 		}
 	}
 
+	if d.KillAfter < 0 {
```

Comment:
Should update the error message to be more explicit "killAfter must not be negative".